lib/python/qmk/path.py

```diff
@@ -13,23 +14,24 @@ def keymap(keyboard):
         keyboard
             The name of the keyboard. Example: clueboard/66/rev3
     """
-    for directory in ['.', '..', '../..', '../../..', '../../../..', '../../../../..']:
-        basepath = os.path.normpath(os.path.join('keyboards', keyboard, directory, 'keymaps'))
+    keyboard_folder = Path('keyboards') / keyboard
 
-        if os.path.exists(basepath):
-            return basepath
+    for i in range(5):
+        if (keyboard_folder / 'keymaps').exists():
+            return (keyboard_folder / 'keymaps').resolve()
 
-    logging.error('Could not find keymaps directory!')
+        keyboard_folder = keyboard_folder / '..'
```

Comment:
Is there a reason this is not
```suggestion
        keyboard_folder = keyboard_folder.parent
```
?